homeassistant/components/workday/binary_sensor.py

```diff
@@ -67,10 +72,15 @@ def valid_country(value: Any) -> str:
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Set up the Workday sensor."""
     add_holidays = config.get(CONF_ADD_HOLIDAYS)
-    remove_holidays = config.get(CONF_REMOVE_HOLIDAYS)
+    remove_holidays = config.get(CONF_REMOVE_HOLIDAYS, [])
```

Comment:
This should be handled automatically be the schema validation, but it seemed easier to change it here than to adjust the tests.
I'm happy to adjust the tests instead if so desired.